ls1intum/Artemis
# [Bugfix] Fix Alert position on Lecture pages

### Checklist
#### General



### Motivation and Context


The Alerts were not placed correctly on lecture pages. In particular, it was different to the exercise pages, where it looked fine. I re-placed the alert text in `course-lecture-details.component.hmtl` to be consistent with `course-exercise-details.component.html`.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Programming Exercise with Complaints enabled

1. Log in to Artemis
2. Navigate to the lecture page
3. Delete or create a post -> see where the alert pops up, should be below the lecture information header
4. Navigate to the exercise page
5. Delete or create a post -> see where the alert pops up, should be below the exercise information header (nothing changed here)

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots
